Marlin/Conditionals_post.h

```diff
@@ -731,7 +731,7 @@
    * Set granular options based on the specific type of leveling
    */
 
-  #define UBL_DELTA  (ENABLED(AUTO_BED_LEVELING_UBL) && ENABLED(DELTA))
+  #define UBL_DELTA  (ENABLED(AUTO_BED_LEVELING_UBL) && (ENABLED(DELTA) || ENABLED(UBL_DELTA_ON_CARTESIAN)))
```

Comment:
If `UBL_DELTA` no longer refers to something delta-specific, then a new name is needed. Meanwhile, `UBL_DELTA_ON_CARTESIAN` is not a good name. Please change. Delta is delta and cartesian is cartesian. Don't "brand name" parts of UBL with a "delta" moniker, if they are not delta-specific.

If the aim is to have all UBL functions working on all systems, like `AUTO_BED_LEVELING_BILINEAR` now does, then we should finish the job and just get rid of all these conditionals.